src/index.js

```diff
@@ -35,20 +31,20 @@ if (utils.isIE11) {
 var error = debug('A-Frame:error');
 var warn = debug('A-Frame:warn');
 
-if (window.document.currentScript && window.document.currentScript.parentNode !==
+if (window.AFRAME_CONFIG.debug.warnIfNotHead && window.document.currentScript && window.document.currentScript.parentNode !==
```

Comment:
Do we need configurable warnings? Can we just check if it's a cordova environment?